GOVERNANCE.md

```diff
@@ -1,36 +1,87 @@
 # Governance
 
-`haskell-cryptography` (herein referred to as the "Organisation") is a GitHub organisation that aims to provide a support structure to open-source maintainers producing cryptography libraries.
+`haskell-cryptography` (herein referred to as the "Organisation") is a GitHub organisation that aims to provide a support structure to open-source maintainers producing cryptography projects.
 
-This document elaborates on administrative and operational aspects of the Organisation.
+The Organisation has the following goals:
+
+* Prevent situations where too few people are supporting too critical projects for the ecosystem.
+* Prevent having a very closed group of people having too much control over every aspect of a project.
+* To encourage and support part-time, niche contributors to the projects.
+* To ensure continuity and stability through documentation, testing and benchmarking.
+* To listen to the needs of the community and industrial users, and lowering the barrier to industrial adoption.
+* To ensure that current security practices are easy for others and maintained whithin the ecosystem.
```

Comment:
I wouldn't mind seeing something about _timely updates_ being a goal, it's all well and good that PRs have been merged but also getting them released is important. (I say this as someone who's terrible at getting releases out)